### ISSUE TITLE
Missing files xdisasmviewoptionswidget

### ISSUE THREAD

**Issue by mrexodia** [horsicq/trkdbg#1]
It looks like some files are not committed to the XDisasmView submodule:

- xdisasmviewoptionswidget.ui
- xdisasmviewoptionswidget.cpp
- xdisasmviewoptionswidget.h

Nice project 🙂

**Comment by horsicq** [comment]
Thanks a lot for the bugreport! Fixed: https://github.com/horsicq/trkdbg/commit/b49a482d94219523a1077de85380cffb9ed9bc66

// Nice project 

Nice to hear it from you. Thanks a lot! :)

But I really don't know when I will finish this debugger. It consists of modules that I use in other projects.

**Comment by mrexodia** [comment]
Yeah I like that you made everything so modular and you have a clean implementation of a table view, very nice!
